components/brand/Features.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <section class="container mx-auto px-6 md:px-8 mt-[83px]">
+    <h3
+      class="uppercase text-primary text-[32px] md:text-5xl text-center font-semibold leading-tight mb-16"
+    >
+      PRODUCT FEATURES
+    </h3>
+    <div class="flex flex-wrap-reverse xl:flex-nowrap items-center justify-center justify-between mb-12 lg:mb-24">
+      <div class="flex flex-col items-center lg:items-start">
+        <p class=" mt-6 mb-8 lg:mb-6 lg:mt-0 text-3xl font-bold text-primary text-center lg:text-left">
+          {{ feature1.fields.title }}
+        </p>
+        <p class="feature-text text-center lg:text-left">
+          {{ feature1.fields.paragraph }}
+        </p>
+        <button class="bg-secondary text-white my-4 py-1.5 px-3 rounded-lg text-xl hover-button-blue">
+          {{ feature1.fields.buttonText }}
+        </button>
+      </div>
+      <img
+        class="w-[342px] lg:w-[625px]"
+        :src="feature1.fields.image.fields.file.url"
+      >
+    </div>
+    <div class="flex flex-wrap xl:flex-nowrap items-center justify-center justify-between mb-12 lg:mb-24">
+      <img
+        class="w-[342px] lg:w-[625px]"
+        :src="feature2.fields.image.fields.file.url"
+      >
+      <div class="flex flex-col items-center lg:items-start">
+        <p class="mt-6 mb-8 lg:mb-6 lg:mt-0 text-3xl font-bold text-primary text-center lg:text-left">
+          {{ feature2.fields.title }}
+        </p>
+        <p class="feature-text text-center lg:text-left">
+          {{ feature2.fields.paragraph }}
+        </p>
+        <button class="bg-secondary text-white my-4 py-1.5 px-3 rounded-lg text-xl hover-button-blue">
+          {{ feature2.fields.buttonText }}
+        </button>
+      </div>
+    </div>
+    <div class="flex flex-wrap-reverse xl:flex-nowrap items-center justify-center justify-between mb-12 lg:mb-24">
+      <div class="flex flex-col items-center lg:items-start">
+        <p class="mt-6 mb-8 lg:mb-6 lg:mt-0 text-3xl font-bold text-primary text-center lg:text-left">
+          {{ feature3.fields.title }}
+        </p>
+        <p class="feature-text text-center lg:text-left">
+          {{ feature3.fields.paragraph }}
+        </p>
+        <button class="bg-secondary text-white my-4 py-1.5 px-3 rounded-lg text-xl hover-button-blue">
+          {{ feature3.fields.buttonText }}
+        </button>
+      </div>
+      <img
+        class="w-[342px] lg:w-[625px]"
+        :src="feature3.fields.image.fields.file.url"
+      >
+    </div>
+    <div class="flex flex-wrap xl:flex-nowrap items-center justify-center justify-between mb-12 lg:mb-24">
```

Comment:
Same suggestion as before

```suggestion
    <div class="flex flex-wrap-reverse xl:flex-nowrap items-center justify-center xl:justify-between mb-12 lg:mb-24">
```

`justify-center` was being overridden by `justify-between`, so `justify-center` had no effect here.

However, it will be good to have `justify-center` for the mobile view. Since for screens wider than 1280px we are showing the two columns but only one column for narrower screens, it will be good to center that content for these narrow screens.